### DISCUSSION TITLE
The last updated time displayed on the page is always Thursday, January 1, 1970 08:00:00

### DISCUSSION THREAD

**Discussion by Anh-bro** [vuejs/vitepress#3723] [Q&A] [answered]
![图片](https://github.com/vuejs/vitepress/assets/76084443/c97c165b-ebcd-4ce9-9870-7dd064ad9aaa)
The time obtained using the "git log" command is correct.
![图片](https://github.com/vuejs/vitepress/assets/76084443/21a8c47d-cb76-4722-ae41-a1da526601cd)
The time of all pages is this, whether it is on the local "vitepress dev docs" or on GithubPage.
why is this happening？
The version number of vitepress is 1.0.1.
I can't find where is the function to get the time.

**Comment by brc-dd** [comment]
Please verify if this is returning correct timestamp:

```sh
git log -1 --pretty="%ai" "path/to/your/file.md"
```

**Reply by Anh-bro** [reply, marked as answer]
thank you for your reply!  I found the reason! 
I added "lastUpdated: true" to the frontmatter of the markdown file. 
I create another file without adding this key-value pair. After submitting, the correct time is displayed on the page. 
Moreover, the last updated timestamp obtained using "useData" is correct. 
It is because "lastUpdated: true" is added to frontmatter that the time on the page is displayed incorrectly.
Is this a bug? It seems to me that the description in the documentation is not accurate:
https://vitepress.dev/reference/frontmatter-config#lastupdated
